sdk/storage/azure-storage-files-datalake/src/datalake_file_system_client.cpp

```diff
@@ -207,11 +229,31 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     blobOptions.AccessConditions.IfUnmodifiedSince = options.AccessConditions.IfUnmodifiedSince;
     blobOptions.AccessConditions.LeaseId = options.AccessConditions.LeaseId;
     auto result = m_blobContainerClient.Delete(blobOptions);
-    Models::FileSystemDeleteResult ret;
-    return Azure::Core::Response<Models::FileSystemDeleteResult>(
+    Models::DeleteFileSystemResult ret;
+    ret.Deleted = true;
```

Comment:
Isn't this `true` already?

Reply:
Yes, but not relying on the default initialized value is a good practice as far as I'm concerned.